walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBPKWIOcOGpjrIfWbDlv0yrh070mH3dFBw"></script>
```

Comment:
Can I steal your API key and charge your account instead of mine ? :)

Reply:
oops should I have hidden this key somewhere?

Reply:
Good question, but idk. Do they hide it in the guide? If you are not sure - you could ask Froyo chat about it (just in case). If you don't get answer until tomorrow - it's fine then.

Reply:
They just tell us to put the script at the top of the html page, but that does seem unsafe so I think it might be better to hide it in the js file? I'll ask on froyo